gdk-sys/build.rs

```diff
@@ -92,3 +98,19 @@ fn find() -> Result<(), Error> {
         Err(err) => Err(err),
     }
 }
+
+#[cfg(not(feature = "dox"))]
+fn check_features() {
+    // The pkg-config file defines a `targets` variable listing the
+    // various backends that gdk was compiled for.
+    // We extract that and create gdk_backend="x11" and the like
+    // as configuration variables.
+    // For reference, the backend set at time of writing consists of:
+    // x11 win32 quartz broadway wayland
+    if let Ok(targets) = pkg_config::get_variable(PKG_CONFIG_PACKAGE, "targets") {
+        println!("cargo:backends={}", targets);
+        for target in targets.split_whitespace() {
+            println!("cargo:rustc-cfg=gdk_backend=\"{}\"", target);
```

Comment:
this not exported as  `DEP_GDK_GDK_BACKENDS` ?

Reply:
No, it's exported as `DEP_GDK_BACKENDS`

Reply:
I felt the repetition of GDK wasn't worth while

Reply:
In case i was confusing... line 111 causes dependencies to have `DEP_GDK_BACKENDS` in the environment of their `build.rs` run.  the `gdk_backend={}` stuff permits (in the future) the use of `'[cfg(gdk_backend="x11")]` attributes in the gdk-sys code.

Reply:
I was also confused, missed line 111. All Ok.